internal/api/ui/login/renderer.go

```diff
@@ -330,15 +340,28 @@ func (l *Login) getUserData(r *http.Request, authReq *domain.AuthRequest, title
 	return userData
 }
 
-func (l *Login) getBaseData(r *http.Request, authReq *domain.AuthRequest, title string, errType, errMessage string) baseData {
-	lang, _ := l.renderer.ReqLang(l.getTranslator(r.Context(), authReq), r).Base()
+func (l *Login) getBaseData(r *http.Request, authReq *domain.AuthRequest, titleI18nKey string, descriptionI18nKey string, errType, errMessage string) baseData {
+	translator := l.getTranslator(r.Context(), authReq)
+	
+	title := ""
+	if (titleI18nKey != "") {
+		title = translator.LocalizeWithoutArgs(titleI18nKey)
+	}
+
+	description := ""
+	if (titleI18nKey != "") {
+		description =  translator.LocalizeWithoutArgs(descriptionI18nKey)
+	}
+	
+	lang, _ := l.renderer.ReqLang(translator, r).Base()
```

Comment:
i'd just pass the i18n key to the html and let i translate there

Reply:
hmm on a second thought, checking for empty strings saves us possible warnings

but then you will need to check for the correct key `if descriptionI18nKey != ""` and please omit the brackets

Reply:
changed checks 👍